aws/table_aws_region.go

```diff
@@ -131,6 +145,48 @@ func getAwsRegion(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateDat
 	return nil, nil
 }
 
+func getDefaultEBSVolumeEncryption(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getDefaultEBSVolumeEncryption")
+	data := h.Item.(*ec2.Region)
+
+	// Returning false for disabled regions to avoid permission denied error AuthFailure
+	if(*data.OptInStatus == "not-opted-in"){
```

Comment:
@sayan133 Can you please format the file?
```suggestion
	if (*data.OptInStatus == "not-opted-in") {
```